satpy/etc/readers/native_msg.yaml

```diff
@@ -8,9 +8,7 @@ reader:
 file_types:
   native_msg:
     file_reader: !!python/name:satpy.readers.native_msg.NativeMSGFileHandler ''
-    file_patterns: ['{satid:4s}-{instr:4s}-MSG{product_level:2d}-0100-NA-{processing_time1:%Y%m%d%H%M%S.%f}000Z-{processing_time2:%Y%m%d%H%M%S}-{order_id:d}.nat',
-                    '{satid:4s}-{instr:4s}-MSG{product_level:2d}-0100-NA-{processing_time1:%Y%m%d%H%M%S.%f}000Z-NA.nat',
-                    '{satid:4s}-{instr:4s}-MSG{product_level:2d}-0100-NA-{processing_time1:%Y%m%d%H%M%S.%f}000Z-{order_id:d}.nat']
+    file_patterns: ['{satid:4s}-{instr:4s}-MSG{product_level:2d}-0100-NA-{processing_time1:%Y%m%d%H%M%S.%f}000Z-{order_id:s}.nat']
```

Comment:
This doesn't include the first original pattern which had a `processing_time2`. If this file pattern doesn't actually exist (check with person who originally added it) then this is fine.

Reply:
´processing_time2` is not used anywhere, the rest of the filename is now caught under `order_id`.

Reply:
Ok so you expect the processing time to just be "eaten up" by the order id? Ok, I'd rather have two file patterns, but I'm ok with this if @adybbroe and @mraspaud are.

Reply:
yes, exactly. those file parts are not needed.